components/emoji_picker/emoji_picker.tsx

```diff
@@ -105,6 +115,7 @@ const EmojiPicker = ({
 
         setCategoryOrEmojisRows(updatedCategoryOrEmojisRows);
         setEmojiPositionsArray(updatedEmojiPositions);
+        throttledSearchCustomEmoji.current(filter, customEmojisEnabled);
     }, [filter, userSkinTone, shouldRunCreateCategoryAndEmojiRows.current]);
```

Comment:
we also might have to add `customEmojisEnabled` to the dependency list for this useEffect

Reply:
thanks for all the comments. Yes you are right, I added it